src/uct/ib/dc/dc_mlx5.c

```diff
@@ -1287,7 +1290,9 @@ static UCS_CLASS_CLEANUP_FUNC(uct_dc_mlx5_iface_t)
     ucs_list_for_each_safe(ep, tmp, &self->tx.gc_list, list) {
         uct_dc_mlx5_ep_release(ep);
     }
-    uct_dc_mlx5_iface_dcis_destroy(self, self->tx.ndci);
+
+    uct_dc_mlx5_iface_dcis_destroy(self, self->tx.ndci +
+                                         uct_dc_mlx5_iface_keepalive_ndci(self));
```

Comment:
maybe change uct_dc_mlx5_iface_keepalive_ndci() to return uct_dc_mlx5_iface_total_ndci()
because seems in most places we do "self->tx.ndci +  uct_dc_mlx5_iface_keepalive_ndci(self)"